chapter3.md

```diff
@@ -64,26 +64,26 @@ Averaging the AICc weights and distinct abundance values across all sites in a d
 
-<--THIS ENTIRE PARAGRAPH NEEDS TO BE WELL CITED-->
-One key difference between terrestrial and marine systems is that marine systems have been heavily impacted by anthropogenic activities.  While terrestrial systems have also experienced anthropogenic influences, there has been a difference in intensity and in patterns of use and management of taxa, with terrestrial systems [CITATION NEEDED].  Primarily wild populations of consumers are targeted in marine systems, while terrestrial systems focus more on land use for domesticated producers [CITATION NEEDED].  This legacy of marine exploitation and over-exploitation is a distinctly non-neutral influence on the structure of marine species abundance distributions that has the potential to produce a strong signal non-neutral signal in the SAD. In fact, there is an area of research using abundance distributions in marine systems to identify disturbed systems [@gray1979; @patil1982; @warwick1986; @magurran2013]. There are several additional potential explanations for the difference in results between our study and the Connolly 2014 paper, some non-biological (spatial structuring, sampling intensity), others related to biological/ecological differences in the data.
+<--I think this is better, and rewritten to be more precise in meaning.-->
+One key difference between terrestrial and marine systems is the way in which these systems have been differently impacted by anthropogenic activities.  There has been a difference in the historical intensity and patterns of use and management of taxa, with technology more recently advanced to more heavily exploit non-coastal marine systems (e.g., fisheries collapses) [crain2008interactive; @jackson2001historical].  Primarily wild populations of consumers are targeted in marine systems, while terrestrial systems focus more on land use for domesticated producers and consumers [marine, @jackson2001historical; @tittensor2009chapter, terrestrial; haberl2007quantifying].  This legacy of marine exploitation and over-exploitation is a distinctly non-neutral influence on the structure of marine species abundance distributions that has the potential to produce a strong signal non-neutral signal in the SAD. In fact, there is an area of research using abundance distributions in marine systems to identify disturbed systems [@gray1979; @patil1982; @warwick1986; @magurran2013]. There are several additional potential explanations for the difference in results between our study and the Connolly 2014 paper, some non-biological (spatial structuring, sampling intensity), others related to biological/ecological differences in the data.
```

Comment:
Yes, that's what I meant.  Thank you.


Elita Baldridge
Ph.D. candidate
Utah State University
Department of Biology and the Ecology Center

On Thu, Mar 26, 2015 at 11:09 AM, Ethan White <notifications@github.com>
wrote:

> In chapter3.md
> <https://github.com/weecology/sad-comparison/pull/147#discussion_r27227638>
> :
>
> >
> > -<--THIS ENTIRE PARAGRAPH NEEDS TO BE WELL CITED-->
> > -One key difference between terrestrial and marine systems is that marine systems have been heavily impacted by anthropogenic activities.  While terrestrial systems have also experienced anthropogenic influences, there has been a difference in intensity and in patterns of use and management of taxa, with terrestrial systems [CITATION NEEDED].  Primarily wild populations of consumers are targeted in marine systems, while terrestrial systems focus more on land use for domesticated producers [CITATION NEEDED].  This legacy of marine exploitation and over-exploitation is a distinctly non-neutral influence on the structure of marine species abundance distributions that has the potential to produce a strong signal non-neutral signal in the SAD. In fact, there is an area of research using abundance distributions in marine systems to identify disturbed systems [@gray1979; @patil1982; @warwick1986; @magurran2013]. There are several additional potential explanations for the difference in
>  results between our study and the Connolly 2014 paper, some non-biological (spatial structuring, sampling intensity), others related to biological/ecological differences in the data.
> > +<--I think this is better, and rewritten to be more precise in meaning.-->
> > +One key difference between terrestrial and marine systems is the way in which these systems have been differently impacted by anthropogenic activities.  There has been a difference in the historical intensity and patterns of use and management of taxa, with technology more recently advanced to more heavily exploit non-coastal marine systems (e.g., fisheries collapses) [crain2008interactive; @jackson2001historical].  Primarily wild populations of consumers are targeted in marine systems, while terrestrial systems focus more on land use for domesticated producers and consumers [marine, @jackson2001historical; @tittensor2009chapter, terrestrial; haberl2007quantifying].  This legacy of marine exploitation and over-exploitation is a distinctly non-neutral influence on the structure of marine species abundance distributions that has the potential to produce a strong signal non-neutral signal in the SAD. In fact, there is an area of research using abundance distributions in marine sys
>  tems to identify disturbed systems [@gray1979; @patil1982; @warwick1986; @magurran2013]. There are several additional potential explanations for the difference in results between our study and the Connolly 2014 paper, some non-biological (spatial structuring, sampling intensity), others related to biological/ecological differences in the data.
>
> "Primarily wild populations of consumers are targeted in marine systems,
> while terrestrial systems focus more on land use for domesticated producers
> and consumers"
> ->
> "One of the major differences is that in marine systems anthropogenic
> efforts to extract resources have focused primarily on wild populations of
> consumers, while in terrestrial systems these efforts have focused more on
> land use for domesticated producers and consumers. This difference leads to
> direct influences on the species being measured by ecologists in marine
> systems, but only indirect effects in terrestrial systems (e.g., through
> land use change)."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weecology/sad-comparison/pull/147/files#r27227638>.
>